db/models/feedback.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Feedback = db.define('feedback', {
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Sequelize.NOW
+  },
+  // e-mail not validated here because email is not required
+  // email should be checked for validity on front-end
```

Comment:
I did a bit of looking into it but I didn't find an easy way to check that an email was a valid email, without requiring the field--ideally it would be a valid email, or null. I'm sure there's a better way to do this though